From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ricardo Pardini <ricardo@pardini.net>
Date: Thu, 18 Dec 2025 00:59:07 +0100
Subject: WIP: add e24c board model to rock-2-rk3528.c

Picked from Kwiboo's +

- E24C: point to -spi DTB variant for kernel

WIP: board: rockchip: Add Radxa E24C

Features tested on a E24C v1.203 (D4E0S16):
- SD-card boot
- SPI flash boot
- LEDs and buttons
- PCIe/NVMe
- USB host
- USB gadget

Signed-off-by: Jonas Karlman <jonas@kwiboo.se>
Signed-off-by: Ricardo Pardini <ricardo@pardini.net>
---
 board/radxa/rock-2-rk3528/rock-2-rk3528.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/radxa/rock-2-rk3528/rock-2-rk3528.c b/board/radxa/rock-2-rk3528/rock-2-rk3528.c
index 111111111111..222222222222 100644
--- a/board/radxa/rock-2-rk3528/rock-2-rk3528.c
+++ b/board/radxa/rock-2-rk3528/rock-2-rk3528.c
@@ -17,6 +17,7 @@ static const struct board_model board_models[] = {
 	{ 63, 278, "rockchip/rk3528-rock-2a.dtb" },
 	{ 291, 392, "rockchip/rk3528-radxa-e20c.dtb" },
 	{ 519, 733, "rockchip/rk3528-rock-2f.dtb" },
+	{ 745, 846, "rockchip/rk3528-radxa-e24c-spi.dtb" },
 };
 
 static const struct board_model *get_board_model(void)
-- 
Armbian

